landoapi/validation.py

```diff
@@ -35,3 +36,8 @@ def parse_landing_path(landing_path: list[dict]) -> list[tuple[int, int]]:
             f"The provided landing_path was malformed.\n{str(e)}",
             type="https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400",
         )
+
+
+def is_valid_email(email: str) -> bool:
+    """Given a string, determines if it is a valid email."""
```

Comment:
You can probably put the regex pattern here, and describe in more detail in the docstring what it's doing.